src/components/PriceBar/BudgetBoard.tsx

```diff
@@ -64,32 +68,37 @@ const BudgetBoard = ({ useTotalPrice }: IBudgetBoardprops) => {
   return (
     <Center
       width="50%"
-      justifyContent={{ base: "flex-start", lg: "flex-end" }}
+      justifyContent={{ base: "space-around", lg: "flex-end" }}
       paddingLeft={{ base: "10px", lg: "0px", xl: "0px" }}
       borderLeft="1px solid #ABABAD"
       alignItems="center"
       color="brand.primary"
     >
-      <Center alignItems="baseline">
-        <Text fontSize={{ base: "xs", lg: "sm" }} fontWeight="600" marginLeft="50px">
-          Estimated Budget
-        </Text>
-        <Text
-          fontSize={{ base: "xs", lg: "sm" }}
-          fontWeight={{ base: "700", lg: "800" }}
-          marginLeft="8px"
+      <Flex flexDirection="column" justifyContent="center" width="40%" color="fontcolor.lightDark">
+        <Flex justifyContent="space-around" fontSize="12px">
+          {" "}
+          <Text marginLeft="20px">Quotation</Text>
+          <Text marginLeft="5px">
+            {useTotalPrice === "0.00" ? "Loading..." : formatCurrency(useTotalPrice)}
+          </Text>
+        </Flex>
+        <Flex
+          justifyContent="space-around"
+          fontSize={{ base: "14px", lg: "20px" }}
+          fontWeight={{ base: "800", lg: "800" }}
         >
-          From
-        </Text>
-        <Text
-          fontSize={{ base: "md", lg: "lg" }}
-          fontWeight={{ base: "700", lg: "800" }}
-          marginLeft="5px"
-          marginRight="20px"
-        >
-          ${useTotalPrice === "0.00" ? "Loading..." : useTotalPrice}
-        </Text>
-      </Center>
+          <Center>
+            <BsQuestionCircle />
```

Comment:
why need to add this icon?

Reply:
well, I saw it was on the Figma UI.
<img width="309" alt="Screen Shot 2022-11-23 at 11 19 50 pm" src="https://user-images.githubusercontent.com/85828920/203545420-4f7e56ce-d424-4dbe-926c-c673ca35be08.png">